### ISSUE TITLE
Error in LMDB function mdb_env_open Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by northeastsquare** [eladhoffer/ImageNet-Training#3]
Hello, I get this error, How shoud I do?

silva@silva-XPS-8300:~/src/ImageNet-Training$ th Main.lua
[program started on Fri May  8 16:58:21 2015]
[command line arguments]
LR  0.01
LRDecay 0
network ./Models/AlexNet
save    /home/silva/src/ImageNet-Training/Results/FriMay816:58:202015
augment true
load  
nGPU    1
modelsFolder    ./Models/
type    cuda
momentum    0.9
devid   1
batchSize   128
threads 8
optimization    sgd
shuffle false
weightDecay 0.0005
epoch   -1
bufferSize  1280
[----------------------]
==> Network
nn.Sequential {
  [input -> (1) -> (2) -> output](1): nn.Sequential {
    [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> (11) -> (12) -> (13) -> output](1): cudnn.SpatialConvolution
    (2): cudnn.ReLU
    (3): cudnn.SpatialMaxPooling
    (4): cudnn.SpatialConvolution
    (5): cudnn.ReLU
    (6): cudnn.SpatialMaxPooling
    (7): cudnn.SpatialConvolution
    (8): cudnn.ReLU
    (9): cudnn.SpatialConvolution
    (10): cudnn.ReLU
    (11): cudnn.SpatialConvolution
    (12): cudnn.ReLU
    (13): cudnn.SpatialMaxPooling
  }
  (2): nn.Sequential {
    [input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> output](1): nn.View
    (2): nn.Dropout
    (3): nn.Linear(9216 -> 4096)
    (4): nn.Threshold
    (5): nn.Dropout
    (6): nn.Linear(4096 -> 4096)
    (7): nn.Threshold
    (8): nn.Linear(4096 -> 1000)
    (9): nn.LogSoftMax
  }
}
==> Loss
nn.ClassNLLCriterion

Epoch 1

Error in LMDB function mdb_env_open :   No such file or directory
Segmentation fault (core dumped)

the Config.lua:
local ImageNetClasses = torch.load('./ImageNetClasses')
for i=1001,#ImageNetClasses.ClassName do
    ImageNetClasses.ClassName[i] = nil
end

return
{
    TRAINING_PATH = '/media/silva/000075C60000073B/ILSVRC2012_img_train.tar',
    VALIDATION_PATH = '/media/silva/000C9D98000CBA6C/val/',
    VALIDATION_DIR = './Data/ValidationCache/',
    TRAINING_DIR = './Data/TrainingCache/',
    ImageSize = 256,
    SampleSize = {3,256,256},
    ValidationLabels = torch.load('./ValidationLabels'),
    ImageNetClasses = ImageNetClasses,
    DataMean = 118.380948,
    DataStd = 61.896913
}

**Comment by eladhoffer** [comment]
You should extract the tar file and create an LMDB record

**Comment by northeastsquare** [comment]
I use titian x, alex ok, but overfeat , system crash;fbcunn imagenet example , result is same, alex ok, but overfeat , system crash whether cudnn or fbcunn. How can it be?

**Comment by eladhoffer** [comment]
Is it the same error message?

**Comment by northeastsquare** [comment]
Where can I see error message?

**Comment by northeastsquare** [comment]
After printing:
==> Loss 
 nn.ClassNLLCriterion 
Epoch 1

Then System crash

**Comment by northeastsquare** [comment]
Both  ImageNet-Training and fbcunn imagenet example, The beginning of epoch 1 ,crash

**Comment by northeastsquare** [comment]
system crash, I mean computer suddenly restart.

**Comment by northeastsquare** [comment]
Reinstall NVIDIA-Linux-x86_64-346.59.run and cuda, solve it.Thank you elahoffer for your reply.

**Comment by waynewhu** [comment]
hello, I met the same problem. I just run the TestLMDB.lua code, then it gives the same error message. what should I do?
